cmd/snap/cmd_snap_op.go

```diff
@@ -211,6 +212,28 @@ func (x *channelMixin) setChannelFromCommandline() error {
 	return nil
 }
 
+// show what has been done
+func showDone(names []string, op string) error {
+	cli := Client()
+	snaps, err := cli.List(names)
+	if err != nil {
+		return err
+	}
+
+	for _, snap := range snaps {
+		channelStr := ""
+		if snap.Channel != "" {
+			channelStr = fmt.Sprintf(" (%s)", snap.Channel)
+		}
+		developerStr := ""
+		if snap.Developer != "" {
+			developerStr = fmt.Sprintf(" from '%s'", snap.Developer)
+		}
+		fmt.Fprintf(Stdout, "%s%s %s%s %s\n", snap.Name, channelStr, snap.Version, developerStr, op)
```

Comment:
This will be impossible to i18n correctly. Could we switch on op and do explicit prints:

```
switch op {
    case "install":
        fmt.Fprintf(Stdout, "%s%s %s%s installed\n", snap.Name, channelStr, snap.Version, developerStr)
    case "remove":
        // ...
    case "refresh":
        // ...
    case "try":
        // ...
    case "enable":
        // ...
    case "disable":
        // ...
    case "revert":
        // ...
}
```